Customizing/global/plugins/Services/Repository/RepositoryObject/ReportExample/classes/class.ilObjReportExampleAccess.php

```diff
@@ -0,0 +1,55 @@
+<?php
+
+require_once 'Services/ReportsRepository/classes/class.ilObjReportBaseAccess.php';
+
+
+class ilObjReportExampleAccess extends ilObjReportBaseAccess {
+
+/**
+* Checks wether a user may invoke a command or not
+* (this method is called by ilAccessHandler::checkAccess)
+*
+* Please do not check any preconditions handled by
+* ilConditionHandler here. Also don't do usual RBAC checks.
+*
+* @param        string        $a_cmd                command (not permission!)
+* @param        string        $a_permission        permission
+* @param        int                $a_ref_id                reference id
+* @param        int                $a_obj_id                object id
+* @param        int                $a_user_id                user id (default is current user)
+*
+* @return        boolean                true, if everything is ok
+*/
```

Comment:
Einrückung. Vielleicht @inheritdocs verwenden und nur in Basisklasse Docu schreiben.